### ISSUE TITLE
activate.bat totally doesn't work with paths

### ISSUE THREAD

**Issue by remram44** [conda/conda-env#59]
```
>conda create -p D:\myenv python=2.7
...
Proceed ([y]/n)? y
...
#
# To activate this environment, use:
# > activate D:\myenv
#

>activate D:\myenv
No environment named "D:\myenv" exists in C:\Anaconda\envs
```
[In activate.bat](https://github.com/conda/conda-env/blob/616e3ddc6380f781e4dd586a0499676fcf0a8cf3/bin/activate.bat#L23):
```batch
if exist "%ANACONDA_ENVS%\%CONDA_NEW_ENV%\conda-meta" goto skipmissingenv
```

**Comment by tswicegood** [comment]
Hi @remram44.  Yeah, we're definitely aware of that and it's unfortunately never worked like that on Windows (even back in the pre-`conda-env` days).  It's definitely on our radar as something to fix so it has the same level of features as the OS X/Linux implementations.  I've added the labels bug and enhancement because this sits between -- it's technically a bug because it's a missing feature, but it was never implemented so it's kind of an enhancement.

---

One question, if you don't mind my asking.  What's your use case for specifying an explicit path instead of using a named environment?  I personally don't use explicit paths and have found the named environments (`--name foo` or `-n foo`) pretty convenient, so I'm always interested in how people are using conda environments differently than I am.

**Comment by remram44** [comment]
I took a look at a bunch of stackoverflow questions and made the changes to the batch scripts. Not sure why you were holding out on this, did I miss something? In any case, someone should proofread very carefully.

----

Using an explicit path is the first thing that came to my mind. Why would I want to use global states if I can have my environments right next to my code? This is not what Vagrant or virtualenv do. I guess the annoying part is that `conda create` doesn't mention that this doesn't work at the moment.

**Comment by tswicegood** [comment]
@remram44 -- thanks for submitting PR #60.  I'll take a look as soon as I can.  I'm pretty strapped for the rest of the day, but should be able to get to it early next week.  The reason we haven't applied any of the various fixes is just a matter of having the bandwidth to test all of the various possible routes across all Windows versions that we support (we go back to Windows XP!).

---

That makes sense to some degree.  It's definitely a departure from virtualenv, but not virtualenvwrapper and not Vagrant.  I think of a named conda environment more like a Vagrant VM or Docker Image.  In Vagrant you have the `Vagrantfile` and then just enough local information to be able to reference the running VM which has data stored elsewhere on the system.  Likewise, you might have a `Dockerfile` with your code, but you always reference the Image when working with Docker which stores the information on the host.  Named environments are tracked by Conda and you can activate them to run your code inside them.  Just like you wouldn't commit a Docker Image or Vagrant VM you shouldn't commit your entire Conda environment, just the `environment.yml` file.

I do agree we need better messaging around how to use these features and how to think about them.  Your input definitely helps me understand how you approached it.

**Comment by remram44** [comment]
Ran into this again today. Is conda-env dead or am I just on some blacklist? This is really frustrating. Anyone please say anything.

**Comment by wulmer** [comment]
I had patched the old conda (from the pre conda-env days) and had a pull request ready (https://github.com/conda/conda/pull/913), but before @asmeurer could merge it, all those activate scripts had been moved to conda-env.
Since then, my pull request is just waste and all the work lost :-(
I would work on activate again, but currently, I can't even make the conda-env tests work as I am still under Anaconda 2.1.
This effectively prevents me from working on conda-env and the activate problem.
But I also can't switch to 2.3 before not the activate problem is solved, because I am working with a lot of environments in non-standard locations (locally patched with my old fixes).
 -> Deadlock.
